src/os/MDNS.cxx

```diff
@@ -41,6 +41,7 @@
 void mdns_publish(const char *name, const char *service, uint16_t port) __attribute__ ((weak));
```

Comment:
These commits to MDNS.cxx, MDNS.hxx, and SocketClient.hxx do not belong in this branch.  I may need some Git help to move them over to the proper branch "bakerstu-simplelink-mdns".